### PR TITLE
Theme cleanup

### PR DESCRIPTION
**Pull request by NicolasSimmonds**
The main changes are on the theme and Avatar and StatusMessage components.

Let me know if there are other components where you think the ColorStyles object can be used.

### CODE REVIEW

File: modules/cactus-web/src/Avatar/Avatar.tsx [hunk 1]
```diff
@@ -83,13 +83,17 @@ const usageMap: UsageMap = {
 }
 
 const variant = (props: AvatarProps) => {
-  const { status } = props
+  const { status, type } = props
 
-  if (status !== undefined) {
+  if (status !== undefined && type === 'alert') {
     return css`
       background: ${avaColor};
```
Comment (wilysword) [inline modules/cactus-web/src/Avatar/Avatar.tsx:90]: Pretty sure this is causing a problem, if you look at the snapshot it says things like `background: background-color:hsl(0,0%,90%);`, which is bad CSS. I think you can probably just change it to
```
return avaColor
```

File: modules/cactus-web/src/Avatar/Avatar.tsx [hunk 2]
```diff
@@ -83,13 +83,17 @@ const usageMap: UsageMap = {
...
     `
-  }
+  } else if (status !== undefined && type === 'feedback') {
+    return css`
+      ${avaColor}
+    `
+  } else return
```
Comment (wilysword) [inline modules/cactus-web/src/Avatar/Avatar.tsx:96]: I'm not sure if we care or not, but in storybook when I select "usage" = alert and check "disabled", the icon vanishes (color goes to white, no background color set).

File: modules/cactus-web/src/Avatar/Avatar.tsx (hunk 1 above)
Comment (NicolasSimmonds) [reply modules/cactus-web/src/Avatar/Avatar.tsx:90]: Yeah you are right, I don't know why thats still there, i thought i changed that. Thanks!

File: modules/cactus-web/src/Avatar/Avatar.tsx (hunk 2 above)
Comment (NicolasSimmonds) [reply modules/cactus-web/src/Avatar/Avatar.tsx:96]: Yup this can be corrected

File: modules/cactus-web/src/Avatar/Avatar.tsx [hunk 3]
```diff
@@ -83,12 +83,19 @@ const usageMap: UsageMap = {
...
+    switch (type) {
+      case 'alert':
+        return css`
+          background: ${avaColor};
+        `
+      case 'feedback':
+        return avaColor
+    }
   }
 }
```
Comment (wilysword) [inline modules/cactus-web/src/Avatar/Avatar.tsx:100]: This whole function duplicates some of the logic from the `avaColor` function. If you changed that function to include the `background` declaration where applicable, you could get rid of one of the two functions.
```
return `background: ${props.theme.colors.status.avatar.error};`
```

File: modules/cactus-theme/src/theme.ts [hunk 4]
```diff
@@ -348,7 +349,7 @@ function fromTwoColor({
         },
         callToAction: {
           backgroundColor: callToAction,
-          color: white,
+          color: callToActionText,
```
Comment (mikesoltow) [inline modules/cactus-theme/src/theme.ts:352]: You probably want to change line 316 as well so they're both referencing the same thing

Comment (NicolasSimmonds) [reply modules/cactus-theme/src/theme.ts:352]: Done

**Pull request merged**